src/ATen/THLongStorageView.h

```diff
@@ -33,6 +40,7 @@ class THLongStorageView {
   }
   long one;
   THLongStorage storage;
+  bool empty_to_null;
```

Comment:
Hmm, I'm not as sure about this approach. Its a little weird to have this as a field. For example, if we pass a ref that is non-empty, but `empty_to_null=True`, is this an expected behavior?

Reply:
Like `zero_dim_to_one` it only applies to empty ArrayRefs.  Unlike `zero_dim_to_one`, we can't handle it in the constructor but have to handle it in `operator THLongStorage*()`. That's why it's stored as a field in the struct.